src/stores/useClientStore.js

```diff
@@ -2,7 +2,7 @@ import { inspect } from 'util'
 import { defineStore } from 'pinia'
 import { encodeAgentId } from '../utils/agent'
 
-const makeUseClientStore = ({ useInterfaceStore, onInit }) => defineStore('client', {
+const makeUseClientStore = ({ useInterfaceStore, onInit, loadKycLevel }) => defineStore('client', {
```

Comment:
```suggestion
const makeUseClientStore = ({ useInterfaceStore, onInit, fetchKycLevel }) => defineStore('client', {
```

I'd call this fetch, with the distinction being "fetch" gets something remotely, "get" gets something locally, and "load" gets something remotely and stores it locally

Reply:
renamed

Reply:
renamed.